src/ios/SimpleCameraPreview.m

```diff
@@ -14,7 +14,7 @@ @implementation SimpleCameraPreview
 - (void) getRatio:(CDVInvokedUrlCommand*)command {
     NSDictionary* config = command.arguments[0];
     @try {
-        if (config[@"targetSize"] != [NSNull null]) {
+        if (config[@"targetSize"] != [NSNull null] && ![config[@"targetSize"] isEqual: @"null"]) {
```

Comment:
https://developer.apple.com/documentation/objectivec/nsobject/1393843-isnotequal

we can try this.